apps/frontend/components/catalog/AddRepository/index.vue

```diff
@@ -167,7 +167,7 @@ const { $schemaService } = useNuxtApp() as any;
 
 const repositoryStore = useRepositoryStore();
 const activityStore = useActivityStore();
-const { aiUiEnabled, availableSchemas } = storeToRefs(useConfigStore());
+const { availableSchemas, parsed } = storeToRefs(useConfigStore());
```

Comment:
Can we rename it here to config or parsedConfig to be a bit more obvious in template